prov/efa/test/efa_unit_test_ep.c

```diff
@@ -543,20 +542,25 @@ void test_efa_rdm_ep_getopt(struct efa_resource **state)
 		FI_OPT_EFA_SENDRECV_IN_ORDER_ALIGNED_128_BYTES,
 		FI_OPT_EFA_WRITE_IN_ORDER_ALIGNED_128_BYTES
 	};
+	size_t	num_opt_names = sizeof(opt_names) / sizeof(int);
 
 	efa_unit_test_resource_construct(resource, FI_EP_RDM);
 
-	/* use a too small opt_len */
-	opt_len = 0;
-	for (i = 0; i < 9; i++) {
-		ret = fi_getopt(&resource->ep->fid, FI_OPT_ENDPOINT, opt_names[i], &opt_val, &opt_len);
-		assert_int_equal(ret, -FI_ETOOSMALL);
+	for (i = 0; i < num_opt_names; i++) {
```

Comment:
Nit: `i < num_opt_names` is a comparison between a signed and unsigned integer type. Not necessarily an issue in this scenario (both values will be >= 0 for each iteration), but it's worth pointing out. Changing `i` to `size_t` is the easy fix here.

Reply:
Fixed